take-your-first-steps-with-rust/understand-common-concepts/includes/7-knowledge-check.md

```diff
@@ -0,0 +1,134 @@
+# Knowledge check
+
+In this unit your challenge is to fix the missing parts of each exercise's code to make them compile.
```

Comment:
I see about what your question was about the knowledge check including exercises. Knowledge checks must be multiple choice questions, so these exercises would need to be included as units instead.

Perhaps the exercises could go right after the units where the concept is introduced. For example:
- 3-structs-and-enums.md
- 4-exercise-structs-and-enums.md
- 5-functions.md
- 6-collection-types.md
- 7-exercise-indexing.md

One thing to keep in mind is that the exercises must also include the solution where learners can find it as well.